yeast-react-renderer/src/helpers/diff.ts

```diff
@@ -72,14 +72,14 @@ export function getDiffRenderData(diffNode: YeastChild): DiffRenderData | undefi
 						React.Fragment,
 						{},
 						React.createElement('span', { className: `${modifiedClassPrefix}old` }, diffNode[prop].substring(0, pivot)),
-						React.createElement('span', { className: `${modifiedClassPrefix}new` }, diffNode[prop].substring(pivot, diffNode[prop.length]))
+						React.createElement('span', { className: `${modifiedClassPrefix}new` }, diffNode[prop].substring(pivot + 1, diffNode[prop.length]))
```

Comment:
Easy fix! Are there any test cases or documents that should be updated to catch this?

Reply:
Sure, just added a test